packages/mcp/src/server.ts

```diff
@@ -564,4 +796,28 @@ export class MCPServer extends MCPServerBase {
       throw error instanceof Error ? error : new Error(`Execution of tool '${toolId}' failed: ${String(error)}`);
     }
   }
+
+  /**
+   * Checks if any resources have been updated.
+   * If the resource is subscribed to by clients, an update notification will be sent.
+   */
+  public async notifyResourcesUpdated({ uri }: { uri: string }): Promise<void> {
```

Comment:
same as the client we should do `MCPServer.resources.notifyUpdated()`, `resources.notifyListChanged()`, etc